tests/unit_tests/test_wave_03.py

```diff
@@ -131,7 +131,9 @@ def test_swap_items_from_their_empty_returns_false():
 
     result = fatimah.swap_items(jolie, item_b, nobodys_item)
 
-    raise Exception("Complete this test according to comments below.")
+    assert result is False
```

Comment:
👍 Using `is` ensures the result isn't just falsy, and isn't even `0`. It's actually `False`.

Reply:
Though we don't expect the inventories to change their contents if a swap doesn't occur, it can still be useful to include that level of detail in a test or two just as an additional check. We could check that Fatimah's inventory still has the expected 3 items, while Jolie's is still empty.